contributing_to_docs/term_glossary.adoc

```diff
@@ -148,6 +148,39 @@ Usage: cluster
 The collection of controllers, pods, and services and related DNS and networking
 routing configuration that are defined on the system.
 
+''''
+=== Cluster API
+
+Usage: Cluster API
+
+Cluster API is a Kubernetes subproject focused on providing declarative APIs and tooling to simplify provisioning, upgrading, and operating multiple Kubernetes clusters.^[link:https://github.com/kubernetes-sigs/cluster-api?tab=readme-ov-file#-what-is-the-cluster-api[1]]^
+For more information, see the link:https://github.com/kubernetes-sigs/cluster-api[Cluster API project repository].
+
+Depending on the platform and version support status, Cluster API replaces Terraform in the OpenShift installation process and can be used to manage machines.
```

Comment:
I got feedback with using Terraform in our docs. Maybe mentioning here that Terraform should not be mentioned in our docs? A PM could confirm this. Maybe this is too out of scope for this update?

Reply:
This is a resource to help writers understand what we write about, not user documentation. The mention of Terraform is here because that is a likely case in which writers will encounter this terminology.

We actually do talk about Terraform in this context in the 4.16 rel notes as well :) 
https://docs.openshift.com/container-platform/4.16/release_notes/ocp-4-16-release-notes.html#ocp-4-16-installation-and-update-aws-capi_release-notes

Reply:
NP. Just wanted to share what was mentioned to me at a PM level when covering Oracle Cloud Infra content. Sometimes when one sees a term in one context they might assume it's OK in another context.

Reply:
Oh yeah I totally get it - that's why I shared the RN instance as another situation :)